packages/gestalt-codemods/40.0.0/__testfixtures__/rename-story-pin-icon-to-idea-pin.input.js

```diff
@@ -0,0 +1,14 @@
+// @flow strict
+import { Box, Icon, IconButton, Pog, Button, Icon as RenamedIcon } from 'gestalt';
+
+export default function TestBox() {
+  return (
+    <Box>
+      <Icon icon='story-pin' size={16}/>
```

Comment:
We should probably also cover ternaries and variables

Reply:
DONE